orc8r/cloud/go/obsidian/handler.go

```diff
@@ -227,7 +229,16 @@ func GetNetworkId(c echo.Context) (string, *echo.HTTPError) {
 	if nid == "" {
 		return nid, NetworkIdHttpErr()
 	}
-	return nid, CheckNetworkAccess(c, nid)
+	sanNid := sanitizeString(nid)
+	return sanNid, CheckNetworkAccess(c, sanNid)
+}
+
+func sanitizeString(strString string) string {
+	strSanitizedString := html.EscapeString(strString)                     //escape special charatchters in HTML text
+	strSanitizedString = strings.Replace(strSanitizedString, "\r", "", -1) //remove line breaks
+	strSanitizedString = strings.Replace(strSanitizedString, "\n", "", -1)
+	strSanitizedString = strings.Join(strings.Fields(strSanitizedString), " ") //remove extra whitespace
+	return strSanitizedString
```

Comment:
what about adding to orc8r utils [here](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=orc8r%2Flib%2Fgo%2Futil%2Futil.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=163&start_col=0&end_row=163&end_col=0). You can add an extra [test case](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=orc8r%2Flib%2Fgo%2Futil%2Futil_test.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=28&start_col=0&end_row=28&end_col=0) to make sure the format is what you expected and it doesn't break anything